### PR TITLE
regression fix - show users belonging to group

### PR DESCRIPTION
**Pull request by cheapsteak**
was showing all users again in the `groups/:groupId/users` list

### CODE REVIEW

File: src/services/getUsers.ts [hunk 1]
```diff
@@ -8,11 +8,11 @@ import dummyUsers from './dummyData/users';
 export const getUsers = ({
   offset = 0,
   limit = 20,
-  query,
-  sortField,
-  sortOrder,
-  groupsId,
-  appsId,
+  query = '',
```
Comment (Jephuff) [inline src/services/getUsers.ts]: Do we want an empty string for these? _.isNil won't filter them out.

Comment (cheapsteak) [reply src/services/getUsers.ts]: ah crap, typescript was yelling at me for not providing this 'required' attribute

:(

Comment (cheapsteak) [reply src/services/getUsers.ts]: Fixed! defaulting to null instead

File: src/components/Groups/GroupsPage.tsx [hunk 2]
```diff
@@ -147,7 +147,7 @@ const render = props => {
                   initialSortField={RESOURCE_MAP.users.initialSortField}
                   selectedItemId={userId}
                   Component={resource.ListItem}
-                  getData={resource.getList}
+                  getData={() => resource.getList({ groupsId: groupId })}
```
Comment (Jephuff) [inline src/components/Groups/GroupsPage.tsx]: Is getData is called with other options? maybe those should be spread in the getList call.

Comment (cheapsteak) [reply src/components/Groups/GroupsPage.tsx]: Good call

Comment (cheapsteak) [reply src/components/Groups/GroupsPage.tsx]: updated https://github.com/kids-first/kf-drp-auth-ui/pull/72/commits/ee9a626a89d897f199b3cd391039757365650c1d

**Pull request merged**